CHANGELOG.md

```diff
@@ -33,6 +33,12 @@
   ```
   
   In Dart SDK `>=1.25.0` this can be relaxed as `part of` can refer to a path.
+  To opt-in, `GeneratorBuilder` now has a new flag, `requireLibraryDirective`.
```

Comment:
Fwiw I actually like this idea, but it hasn't technically landed yet even at -dev. Maybe we can set the default _based_ on that once 1.25.0 lands officially?